tests/test_types.py

```diff
@@ -733,6 +734,34 @@ class Model(BaseModel):
     ]
 
 
+def test_str_limited_good():
+    max_length = 5
+
+    class Model(BaseModel):
+        v: StrLimited[5]
+
+    m = Model(v='abcde ')
+    assert m.v == 'abcde'
+
+
+def test_str_limited_too_long():
+    max_length = 5
```

Comment:
you can remove this and just use a hard-coded value below, it's not adding anything.